app/src/main/res/xml/appfilter.xml

```diff
@@ -2679,13 +2682,14 @@
     <item component="ComponentInfo{com.capitainetrain.android/com.capitainetrain.android.HomeActivity}" drawable="trainline" />
     <item component="ComponentInfo{com.thetrainline/com.thetrainline.activities.home_screen.SplashScreenActivity}" drawable="trainline" />
 
-    <!-- Transdroid -->
-    <item component="ComponentInfo{org.transdroid.search/org.transdroid.search.gui.SettingsActivity}" drawable="transdroid" />
-    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity}" drawable="transdroid" />
-
     <!-- Transdrone -->
     <item component="ComponentInfo{org.transdroid.lite/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdrone" />
 
+    <!-- Transdroid -->
+    <item component="ComponentInfo{org.transdroid.search/org.transdroid.search.gui.SettingsActivity}" drawable="transdroid"/>
+    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity}" drawable="transdroid"/>
+    <item component="ComponentInfo{org.transdroid.full/org.transdroid.core.gui.TorrentsActivity_}" drawable="transdroid"/>
```

Comment:
I'm not sure underscores are even allowed for class names, are you sure this was right?

Reply:
>  I'm not sure underscores are even allowed for class names

There are plenty of other apps listed in the manifest that use underscores?!

I believe that "SettingsActivity" and "TorrentsActivity_" are the actual correct 2 classes, but I _think_ "TorrentsActivity" was already added by someone else, so I didn't want to remove it, and the fact that there seems to be 2 different naming schenes (i.e. with and without underscore) confuses me a little .  The app manifest is [here](https://github.com/erickok/transdroid/blob/master/app/src/main/AndroidManifest.xml); perhaps, someone can double check my work?

Reply:
It's also confused by the fact there are multiple versions of the app, and also a search plugin that appears as a seperate icon alongside. Currently, for me, the search plugin has the Frost icon, but the main app does not.

Reply:
I'm not sure about why certain ones aren't applying, that's weird.